layouts/partials/head.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <link href="//gmpg.org/xfn/11" rel="profile">
   <meta http-equiv="content-type" content="text/html; charset=utf-8">
-  {{ .Hugo.Generator }}
+  {{ hugo.Generator }}
```

Comment:
If you use .Site.Hugo.Generator it will be backwards compatible.

Reply:
`.Site.Hugo.Generator` seems to work only starting with v0.53.

When I try with v0.52 (or below) I see this:

```
executing "partials/head.html" at <.Site.Hugo.Generator>: can't evaluate field Generator in type *hugolib.HugoInfo
```

Reply:
Odd. Ok, thanks.